db/schema.rb

```diff
@@ -153,18 +153,37 @@
              FROM (application_version
                JOIN application ON ((application_version.application_id = application.id)))
           )
-   SELECT base.id,
-      base.application_type,
-      base.version,
-      base.from_status,
-      base.from_time,
-      base.to_status,
-      base.to_time,
-      (base.from_time)::date AS from_date,
-      (base.to_time)::date AS to_date,
-      GREATEST(EXTRACT(epoch FROM (base.to_time - base.from_time)), (0)::numeric) AS processing_seconds
+   SELECT id,
```

Comment:
can we understand what is going on here please?

Reply:
AFAIU nothing is changing, but I'll audit it when I've got the last bits of work done

Reply:
But why are the semantics changed? They shouldn't be.

Reply:
I don't know why, but I've removed them now anyway.